src/containers/booksList.js

```diff
@@ -1,21 +1,21 @@
-/* eslint-disable react/forbid-prop-types */
+/* eslint-disable react/prop-types */
+/* eslint-disable react/prefer-stateless-function */
```

Comment:
Use a functional component instead, no need to disable the rule.

Reply:
DONE